nvflare/app_common/pt/pt_model_locator.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List
+
+from nvflare.apis.dxo import DXO, DataKind
+from nvflare.apis.event_type import EventType
+from nvflare.apis.fl_context import FLContext
+from nvflare.app_common.abstract.model_locator import ModelLocator
+from nvflare.app_common.abstract.model_persistor import ModelPersistor
+
+
+class PTModelLocator(ModelLocator):
+    def __init__(self, pt_persistor_id: str):
+        """The ModelLocator's job is to find the models to be included for cross site evaluation
```

Comment:
We should not say "... for cross site eval ...". Model locator's job is to local models. This could be useful to many other purposes, even though it is started with the need of cross site val.